city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    def parse(self, response):
+        """   `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping needs.   """
+        meeting_location = ''
+
+        for item in response.css("#content-232764 div.panel-body p"):
+
+            if item.css("p > strong ::text").getall():
+                meeting_location = self._parse_location(item)
+                continue
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=meeting_location,
+                links=self._parse_links(item),
+                source=response.url,
+            )
+            meeting['status'] = self._get_status(meeting)
+            meeting['id'] = self._get_id(meeting)
+            print(str(meeting))
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        item_txt = ''.join(item.css("p::text").getall())
+        if "Annual Meeting" in item_txt:
+            return "Annual Meeting"
+        else:
+            return COMMISSION
+
+    def _parse_classification(self):
+        # This can return COMMISSION generally or COMMITTEE for the committee meetings
```

Comment:
Just a note for when you add committees that this can accept the `title` or just the meeting string as an argument and check if it contains `committee`, returning `COMMITTEE` if it does. Example here https://github.com/City-Bureau/city-scrapers/blob/2a5dfad69057f090e3d52d73a1ae4d992967c731/city_scrapers/spiders/chi_ssa_33.py#L109

Reply:
They have a separate panel for Committees.
![comms](https://user-images.githubusercontent.com/12592910/61103884-c7d84f80-a428-11e9-8297-ed9fb2c5c5e5.JPG)

Reply:
I think for now we ignore that panel since it doesn't seem to be regularly updated and only pull committees from the minutes section

Reply:
I finished this panel just so I could understand how to process two panels at once.

Reply:
I've taken that panel out in my new revision that I haven't checked in yet.